### PR TITLE
feat(1967): add ability to do raw queries. BREAKING CHANGE: pulling in new datastore-base version

### PR DESCRIPTION
**Pull request by InderH**
## Context

Need to add the ability to do raw queries to datastore so queries that are otherwise impossible to do through sequelize can be done.

## Objective

Add a query method to datastore.

## References

https://github.com/screwdriver-cd/screwdriver/issues/1967

## License

I confirm that this contribution is made under the terms of the license found in the root directory of this repository's source tree and that I have the authority necessary to make this contribution on behalf of its copyright owner.

### CODE REVIEW

File: index.js [hunk 1]
```diff
@@ -568,6 +568,34 @@ class Squeakquel extends Datastore {
...
+     * @param  {Object}        config                Configuration object
+     * @param  {Array<Object>} [config.queries]      Map of database type to query
+     * @param  {String}        [config.table]        Table name
+     * @param  {Object}        [config.replacements] Parameters to replace in the query
+     * @param  {Boolean}       [config.rawResponse]  Return raw response without binding to model
+     */
+    _query(config) {
+        const table = this.tables[config.table];
+        const model = this.models[config.table];
+        const query = config.queries.find(q => q.dbType === this.client.getDialect()).query;
```
Comment (DekusDenial) [inline index.js]: if no query matched, throw some meaningful err message

Comment (pritamstyz4ever) [inline index.js]: Probably will need some null checks here

Comment (InderH) [reply index.js]: Added, thank you

Comment (InderH) [reply index.js]: Added, thank you

**Comment by tkyi** [comment]
Will need to take in ^4.0.0 once we bump major in `datastore-base`
https://github.com/screwdriver-cd/datastore-sequelize/blob/master/package.json#L55

**Comment by tkyi** [comment]
@InderH Can you make the change I mentioned above?^
Then this PR will be ready to merge

**Pull request merged**